packages/contract/src/DaimoEphemeralNotes.sol

```diff
@@ -27,8 +27,7 @@ contract DaimoEphemeralNotes {
     event NoteRedeemed(Note note, address redeemer);
 
     // Stablecoin token to use for notes, or 0x0 for USDC.
-    // This allows us to CREATE2-deploy this contract at the same address
-    // on Base and Base Goerli.
+    // This allows us to CREATE2-deploy at the same address on different chains.
```

Comment:
this edit changes the create2 address no, would revert unless you're migrating deployment

Reply:
good call, reverted